cluster-state-service/handler/store/stm_applier.go

```diff
@@ -64,28 +64,6 @@ func (applier STMApplier) applyVersionedRecord(stm concurrency.STM) error {
 	return nil
 }
 
-// applyVersionedRecord adds a new unversioned record to the store.
-// An unversioned record is generated while bootstrapping and reconciliation
-// workflows.
-func (applier STMApplier) applyUnversionedRecord(stm concurrency.STM) error {
```

Comment:
May want to rename the other function `applyVersionedRecord` to `applyRecord` now that there is no differentiation.